protocol/src/main/java/net/md_5/bungee/protocol/packet/ActionBar.java

```diff
@@ -0,0 +1,42 @@
+package net.md_5.bungee.protocol.packet;
+
+import io.netty.buffer.ByteBuf;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import net.md_5.bungee.protocol.AbstractPacketHandler;
+import net.md_5.bungee.protocol.DefinedPacket;
+import net.md_5.bungee.protocol.ProtocolConstants;
+
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class ActionBar extends DefinedPacket
+{
+
+    // Action bar message
+    private String text;
+
+    public ActionBar(String text)
+    {
+        this.text = text;
+    }
+
+    @Override
+    public void read(ByteBuf buf, ProtocolConstants.Direction direction, int protocolVersion)
+    {
+        text = readString( buf );
```

Comment:
Correct max string length for chat string
```suggestion
        text = readString( buf, 262144 );
```

Reply:
Context: https://github.com/SpigotMC/BungeeCord/commit/c96628b72e2cfa46ba12a76a1262c00085fbe99f#r51271134

Reply:
So we should change to use the maxLen for the title/subtitle packets to ?

Reply:
I'd say so, yes.

Reply:
> 
> 
> So we should change to use the maxLen for the title/subtitle packets to ?

Yes, I just doublechecked in 1.17 client decompile